src/helix-ui/elements/HXTabpanelElement.js

```diff
@@ -1,37 +1,23 @@
 import { HXElement } from './HXElement';
-import shadowStyles from './_hx-tabpanel.less';
+import { HXRevealElement } from './HXRevealElement';
 
-const tagName = 'hx-tabpanel';
-const template = document.createElement('template');
-
-template.innerHTML = `
-  <style>${shadowStyles}</style>
-  <slot id="content"></slot>
-`;
-
-export class HXTabpanelElement extends HXElement {
+export class HXTabpanelElement extends HXRevealElement {
```

Comment:
`super()` possibly can be added. Review.

Reply:
Addressed on [L10](https://github.com/rackerlabs/helix-ui/pull/115/files#diff-5f122be9e80961ad7d10146424f5850fR10)